scripts/create_test_subset.py

```diff
@@ -104,19 +103,17 @@ def main(
     pid_sid = papi.get_external_participant_id_to_internal_sample_id(project)
     sample_id_by_participant_id = dict(pid_sid)
 
-    ped_lines = export_ped_file(project, replace_with_participant_external_ids=True)
     if families_n is not None:
-        ped = Ped(ped_lines)
-        families = list(ped.families.values())
-        logger.info(f'Found {len(families)} families, by size:')
-        _print_fam_stats(families)
-        families = random.sample(families, families_n)
-        logger.info(f'After subsetting to {len(families)} families:')
-        _print_fam_stats(families)
-        sample_ids = []
-        for fam in families:
-            for s in fam.samples:
-                sample_ids.append(sample_id_by_participant_id[s.sample_id])
+        fams = fapi.get_families(project)
+        all_families = [family['id'] for family in fams]
+        families = random.sample(all_families, families_n)
```

Comment:
Missing a `_print_fam_stats` call here

Reply:
Ah yes, would have to reimplement that function to work with the new input format. Is the print useful to have?

Reply:
Fixed in the latest :)